raindrop_common.go

```diff
@@ -114,7 +114,7 @@ func search_request(query string, token RaindropToken, collection int, tag strin
 	var result map[string]interface{}
 	var err error
 	client := &http.Client{}
-	request, err := http.NewRequest("GET", "https://api.raindrop.io/rest/v1/raindrops/"+fmt.Sprint(collection)+"/?search=["+tag+"{\"key\":\"word\",\"val\":\""+url.QueryEscape(query)+"\"}]&sort=\""+sorting+"\"", nil)
+	request, err := http.NewRequest("GET", fmt.Sprintf("https://api.raindrop.io/rest/v1/raindrops/%d/?search=%s&sort=%s", collection, url.QueryEscape(query), sorting), nil)
```

Comment:
This removes the `tag` variable, which is passed into this function and modified on L103-105. Does that break any search feature (eg. searching by tag)?

Reply:
The commit raindropio/developer-site@d90a540 replaced the deprecated search parameter syntax with a link to the [Advanced Search Operations](https://help.raindrop.io/using-search/#operators) section of the Search page documentation. Adhering to the guidelines in that section, the deprecated syntax example `/rest/v1/raindrops/0?search=[{"key":"tag","val":"apple"},{"key":"word","val":"ios 13"}]` would be updated as follows: `/rest/v1/raindrops/0?search=#apple #"ios 13"`. Some quick tests of the search in Alfred suggest that the advanced search operators are supported, including the #tag syntax. Other workflow features that I typically use appear to continue to work. Whether this re-enables all the features of the workflow is an open question.

Reply:
Hi!

Thanks for the pull request @roscoe 

It does work well, except that it ignores tags (as @cdzombak suspected), which in practice means that it will work as intended in all situations except if you browse to a tag and try to search the content of it, where it will simply return all results rather then being specific to the current tag.

I will merge this, and then fix the tag issue, and expect to release the update that includes this within a few hours.